docs/streams/developer-guide/config-streams.html

```diff
@@ -231,9 +231,9 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
         <p>Here are the optional <a href="/{{version}}/javadoc/org/apache/kafka/streams/StreamsConfig.html">Streams</a> javadocs, sorted by level of importance:</p>
         <blockquote>
           <div><ul class="simple">
-            <li>High: These parameters can have a significant impact on performance. Take care when deciding the values of these parameters.</li>
-            <li>Medium: These parameters can have some impact on performance. Your specific environment will determine how much tuning effort should be focused on these parameters.</li>
-            <li>Low: These parameters have a less general or less significant impact on performance.</li>
+            <li>High: These are parameters with default value which are most likely not a good fit for production use. It's highly recommended to revisit these parameters for production usage.</li>
+            <li>Medium: The default values of these parameters should work for production for many cases, but it's not uncommon that they are changed, for example to tune performance.</li>
+            <li>Low: It should be rarely necessary to change the default for these parameters. It's only recommended to change them if there is a very specific issue you want to address.</li>
```

Comment:
Many configs have nothing to do with performance. Trying to make the level description more useful / actionable to users.